0xPolygon/polygon-edge
# Implement removing from filter by ws

# Description

This PR fixes the issue with closing ws connection. 

Fixes EDGE-584

# Changes include


# Checklist


## Testing


### Manual tests

Subscribe to newBlockHeadEvent (https://github.com/Aleksao998/SubscribeToNewHeadEvent) and close the connection, you should not see errors (on develop), on this branch no error logs should appear
```
2022-05-30T14:59:00.022+0200 [ERROR] polygon.jsonrpc: Unable to write WS message, writev tcp 127.0.0.1:10002->127.0.0.1:64893: use of closed network connection
2022-05-30T14:59:00.022+0200 [ERROR] polygon.filter: Unable to process flush, writev tcp 127.0.0.1:10002->127.0.0.1:64893: use of closed network connection
```
